bdns/dns.go

```diff
@@ -177,6 +249,14 @@ func NewTestDNSResolverImpl(readTimeout time.Duration, servers []string, stats m
 	return resolver
 }
 
+// LookupIPV6 controls whether the LookupHost function resolves AAAA records.
```

Comment:
Do we intend to keep this feature flag around after turning on IPv6, i.e. for other users than LE? If not this seems overkill and should probably just be replaced with an exposed bool.